README.md

```diff
@@ -125,6 +125,14 @@ String. Path to the intended install location of the payload on the target disk.
 **name**  
 String containing the package name. If this is missing, one is constructed using the name of the package project directory.
 
+If you would like munki-pkg to automatically append a version to each package during build, add `-${version}` to the end of the name.
+
```

Comment:
??? why would anyone want the version added _after_ the pkg extension?

The intended behavior is variable substitution...